Esterni/PianoDiQualifica/res/sections/test/test.tex

```diff
@@ -0,0 +1,290 @@
+		\item il venditore possa contattare il cliente tramite un form.
+	\end{enumerate} \\
+	\valueCTS & Verifica dashboard aliquote venditore & Verificare che:
+	\begin{enumerate}
+		\item il venditore possa gestire le aliquote IVA per la tassazione dei prodotti;
+		\item il venditore possa visualizzare le aliquote IVA già presenti nel sistema;
+		\item il venditore possa aggiungere un'aliquota IVA;
+		\item il venditore possa modificare un'aliquota IVA;
+		\item il venditore possa eliminare un'aliquota IVA già presente nel sistema.
+	\end{enumerate} \\
+	\valueCTS & Verifica dashboard prodotti venditore & Verificare che:
+	\begin{enumerate}
+		\item il venditore possa visualizzare la lista dei prodotti inseriti nel sistema;
+		\item il venditore possa cercare un prodotto presente nel sistema;
+		\item il venditore possa filtrare i prodotti;
+		\item il venditore possa aggiungere un prodotto al sistema dopo aver compilato tutti i campi obbligatori del prodotto;
+		\item il venditore possa modificare un prodotto aggiornando i campi già presenti;
+		\item il venditore possa eliminare un prodotto presente nel sistema;
+		\item il venditore possa selezionare i prodotti da mostrare nella HomePage cliente.
+	\end{enumerate} \\
+	\valueCTS & Verifica dashboard categorie venditore & Verificare che:
+	\begin{enumerate}
+		\item il venditore possa visualizzare tutte le categorie presenti nel sistema;
+		\item il venditore possa aggiungere una categoria di prodotti impostando il nome;
+		\item il venditore possa rimuovere una categoria di prodotti;
+		\item il venditore possa modificare il nome di una categoria già esistente nel sistema.
+	\end{enumerate} \\
+	\valueCTS & Verifica login venditore & Verificare che il venditore possa effettuare il login \\
+	\valueCTS & Verifica login venditore & Verificare che il venditore possa effettuare il logout \\
+	\hline
+	\end{longtable}
+\end{center}
+
+
+\subsection{Tracciamento test di sistema - requisiti}
+
+\resetCTS
+\begin{center}
+	\rowcolors{1}{white}{blue!20}
+	\begin{longtable}{|c|c|}
+	\hline
+	\rowcolor{lighter-grayer}
+	\textbf{Test di sistema} & \textbf{Requisiti} \\
+	\hline
+	\endfirsthead
+	\hline
+	\multicolumn{2}{|c|}{\textit{Continua nella pagina successiva...}} \\
+	\hline
+	\endfoot
+	\endlastfoot
+
+	\hline
+	\valueCTS & R1F1 \\
+	\valueCTS & R1F2 \\
+	\valueCTS & R1F3 \\
+	\valueCTS & R1F4 \\
+	\valueCTS & R1F5 \\
+	\valueCTS & R1F6 \\
+	\valueCTS & R1F7 \\
+	\valueCTS & R1F8 \\
+	\valueCTS & R1F9 \\
+	\valueCTS & R1F10 \\
+
+	\valueCTS & R1F11  \\
+	\valueCTS & R1F12 \\
+	\valueCTS & R1F13 \\
+	\valueCTS & R1F14 \\
+	\valueCTS & R1F15 \\
+	\valueCTS & R1F16 \\
+	\valueCTS & R1F17 \\
+	\valueCTS & R1F18 \\
+	\valueCTS & R1F18 \\
+	\valueCTS & R1F18 \\
+
+	\valueCTS & R1F19 \\
+	\valueCTS & R1F20 \\
+	\valueCTS & R1F21 \\
+	\valueCTS & R1F22, R1F23 \\
+	\valueCTS & R1F24  \\
+	\valueCTS & R1F25 \\
+	\valueCTS & R1F26 \\
+	\valueCTS & R1F27 \\
+	\valueCTS & R1F28 \\
```

Comment:
Non ci son quasi i sottorequisiti, nonostante nei test siano coperti. I sottorequisiti sono da aggiungere qui nel tracciamento

Reply:
includiamo una dicitura in cui si specifica che un test copre anche tutti i sottorequisiti del requisito padre collegato